api.yml

```diff
@@ -17,13 +17,38 @@ tags:
   - name: public
     description: Public APIs facing the Holder and Verifier.
 paths:
-  /credential:
+  /credentials:
     post:
       tags:
         - internal
       summary: Issues a credential and returns it in the response body.
       operationId: issueCredential
       description: Issues a credential and returns it in the response body. Support of this part of the API is REQUIRED for implementations.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/IssueCredentialRequest'
+        description: Parameters for issuing the credential.
+      responses:
+        '201':
+          description: Credential successfully issued!
+          content:
+            application/json:
+              schema:
+                type: object
+                description: The issued credential. This SHOULD include a reference or ID that can be used later for other parts of the API, e.g. changing and retrieving the status of the credential, or refreshing the credential.
+        '400':
+          description: invalid input!
+        '500':
+          description: error!
+  /credentials/createAndIssue:
+    post:
+      tags:
+        - internal
+      summary: Composes and issues a credential and returns it in the response body.
+      operationId: composeAndIssueCredential
```

Comment:
is the operationId usually different than the path? `composeAndIssueCredential` vs `createAndIssue`

Reply:
yes, https://swagger.io/docs/specification/paths-and-operations/

Reply:
This inconsistency was introduced by a very rushed sequence of commits that replaced "create" with "compose" everywhere. Well almost everywhere.

Reply:
I fixed this by making it composeAndIssueCredential

Reply:
@peacekeeper I fixed this one case, I'm not a fan of the word "create" here.

Reply:
Okay we can call it "compose" to avoid misunderstandings..

Reply:
I'm resolving this thread based on your approval @peacekeeper.